ngraph/core/src/op/divide.cpp

```diff
@@ -161,3 +114,8 @@ bool op::v1::Divide::evaluate(const HostTensorVector& outputs, const HostTensorV
     OV_ITT_SCOPED_TASK(itt::domains::nGraphOp, "op::v1::Divide::evaluate");
     return evaluate_divide(inputs[0], inputs[1], outputs[0], get_autob(), is_pythondiv());
 }
+
+shared_ptr<Node> ngraph::operator/(const Output<Node>& arg0, const Output<Node>& arg1)
```

Comment:
I suppose that we can remove this operator.

Reply:
All overloaded operators were removed